lib/metrics/processHandles.js

```diff
@@ -28,19 +28,11 @@ module.exports = (registry, config = {}) => {
 		registers: registry ? [registry] : undefined
 	});
 
-	const updater = config.timestamps
-		? () => {
-				const handles = process._getActiveHandles();
-				updateMetrics(gauge, aggregateByObjectName(handles), true);
-				totalGauge.set(handles.length, Date.now());
-		  }
-		: () => {
-				const handles = process._getActiveHandles();
-				updateMetrics(gauge, aggregateByObjectName(handles), false);
-				totalGauge.set(handles.length);
-		  };
-
-	return updater;
+	return () => {
+		const handles = process._getActiveHandles();
+		updateMetrics(gauge, aggregateByObjectName(handles), false);
```

Comment:
do we need this `updateMetrics` helper now?

Reply:
Yes, its shared code, used by lib/metrics/processRequests.js and lib/metrics/processHandles.js.

I think I can remove the second argument to aggregateByObjectName() though, its always passed `false`.

Reply:
done